src/components/rating/rating.jsx

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var React = require('react');
+
+module.exports = React.createClass({
+  propTypes: {
+    children: React.PropTypes.any,
+    rating: React.PropTypes.number.isRequired,
+    outOf: React.PropTypes.number,
+    blankIcon: React.PropTypes.node
+  },
+
+  render: function() {
+    var $rating = [];
```

Comment:
I presume these variables are prefixed with a dollar to denote they are dom nodes... if this were jQuery I would agree with this as a way to let us know we have access to jQuery methods & props but here it's a little confusing... that said, the fact I could tell is was a DOM node at a glance may speak for itself.